torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -5,10 +5,9 @@
 #ifdef USE_CUDA_NOT_ROCM
 #include <c10/core/DeviceGuard.h>
 #include <c10/cuda/CUDACachingAllocator.h>
-#include <tensorpipe/tensorpipe.h>
 #endif
 
-#include <tensorpipe/core/message.h>
+#include <tensorpipe/tensorpipe.h>
```

Comment:
just to double check, we need to move this out of the `USE_CUDA_NOT_ROCM` scope because we need to access `tensorpipe::CpuBuffer` in non CUDA cases?

Curious, how does the previous version work? Looks like it also accesses `tensorpipe::DeviceType::kCpu`?

Reply:
Right, so we need both `tensorpipe/common/cpu_buffer.h` and (conditionally) `tensorpipe/common/cuda_buffer.h`, and since `tensorpipe/tensorpipe.h` was already being included (in the CUDA case) it seemed more straightforward to just add it in all cases.